### PR TITLE
feat(front): remove inputs after validation/cancel

### LINKED ISSUE #88: Remove interactive segmentation prompts once the mask is created
Capture d’écran 2024-02-02 à 15 29 06

### PR DESCRIPTION
**Pull request by BertrandRenault**
Remove construction inputs after validation/cancel
Also allow "Escape" key along with cancel button
close #88

### CODE REVIEW

**Review by cpvannier — APPROVED** [review]

File: ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte [hunk 1]
```diff
@@ -82,10 +82,16 @@
 
       return [...oldObjects, ...(newObject ? [newObject] : [])];
     });
-    newShape.set({ status: "none" });
+    newShape.set({ status: "none", shouldReset: true });
     canSave.set(true);
     currentTab = "objects";
   };
+
+  async function handleKeyDown(event: KeyboardEvent) {
```
Comment (timothee-LJN) [inline ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte:90]: question : why did you add this function here and not in canvas?

**Review by timothee-LJN — APPROVED** [review]

File: ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte (hunk 1 above)
Comment (BertrandRenault) [reply ui/components/imageWorkspace/src/components/SaveShape/SaveShapeForm.svelte:90]: because newShape is a Writable<Shape> here in SaveShapeForm, but a Shape in Canvas2D, so shpae doesn't have the set function needed here to close the form
